manual/manual_circuit.rst

```diff
@@ -482,28 +507,35 @@ Another notable example that is common to many algorithms and high-level circuit
     circ.add_pauliexpbox(PauliExpBox([Pauli.X, Pauli.Y, Pauli.Y, Pauli.Y], 0.2), [0, 1, 2, 3])
     circ.add_pauliexpbox(PauliExpBox([Pauli.Y, Pauli.X, Pauli.Y, Pauli.Y], -0.2), [0, 1, 2, 3])
 
-Statevectors and Unitaries
---------------------------
+In addition to the box types mentioned above ``pytket`` also supports a :py:class:`ToffoliBox` structure. :py:class:`ToffoliBox` can be used to prepare an arbitrary permutation of the computational basis states using. Within the box this permutation is performed using the classical reversible gateset :math:`\{X,CnX\}`.
```

Comment:
just say "the gateset :math:`\{X,CnX\}`.

Reply:
done